api/candidatesApiClient.ts

```diff
@@ -20,11 +21,23 @@ export async function getCandidates(page: number, pageSize: number): Promise<Lis
     const apiURL = publicRuntimeConfig.API_URL;
     const response = await fetch(`${apiURL}/candidates?page=${page}&pageSize=${pageSize}`);
 
-    if(response.ok){
+    if (response.ok) {
         return await response.json();
-    }
-    else
-    {
+    } else {
         throw Error;
     }
-}
\ No newline at end of file
+}
+
+export async function getCandidateById(cid: any): Promise<Candidate> {
```

Comment:
I think the candidate ID should always be an integer right? Could we enforce that through the type system?

Reply:
I was trying to enforce this but running into difficulty converting the context.query to an int. Kept trying different parsing methods and getting string || string[] can't be number or can't be string or can't be string[].  Will keep looking for a way to change this...

Reply:
ah, got it! a simple 'as' 😅